packages/roosterjs-editor-types/lib/interface/FormatState.ts

```diff
@@ -59,6 +59,11 @@ export interface ElementBasedFormatState {
      */
     isBlockQuote?: boolean;
 
+    /**
+     * Whether the text is in Code block
+     */
+    isCode?: boolean;

```

Comment:
Should it be is**CodeBlock** for congruency with toggle**CodeBlock**?

Not sure if you would want to reserve the shorter `isCode` for inline code:

This is `inline code` formatting vs...

```
code
block
formatting
````

Reply:
I'm ok with isCodeBlock.